autocodesign/autocodesign.go

```diff
@@ -0,0 +1,175 @@
+package autocodesign
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-xcode/autocodesign/devportalclient/appstoreconnect"
+	"github.com/bitrise-io/go-xcode/certificateutil"
+	"github.com/bitrise-io/go-xcode/devportalservice"
+	"github.com/bitrise-io/go-xcode/xcodeproject/serialized"
+)
+
+// Profile ...
+type Profile interface {
+	ID() string
+	Attributes() appstoreconnect.ProfileAttributes
+	CertificateIDs() (map[string]bool, error)
+	DeviceIDs() (map[string]bool, error)
+	BundleID() (appstoreconnect.BundleID, error)
+	Entitlements() (serialized.Object, error)
```

Comment:
```suggestion
	Entitlements() (Entitlement, error)
```

This could use the typealias defined below.

(BTW, is this a single entitlement, or a collection of entitlements? The method name suggests its multiple entitlements)

Reply:
Multiple ones